pkg/config/testdata/TestEmailDiff/local_disabled_remote_enabled.diff

```diff
@@ -22,7 +22,7 @@ diff remote[auth] local[auth]
  [email.template]
  [email.template.confirmation]
  content_path = ""
-@@ -71,13 +71,6 @@
+@@ -66,13 +66,6 @@
```

Comment:
**remark**

I'm not totally sure why those snapshot have differences.

Reply:
coz we added uri and secrets fields in tests, so there are more lines in the config dump.